client/state/sites/actions.js

```diff
@@ -27,13 +27,15 @@ import {
  * Returns an action object to be used in signalling that a site has been
  * deleted.
  *
- * @param  {Number} siteId ID of deleted site
- * @return {Object}        Action object
+ * @param  {Number} siteId  ID of deleted site
+ * @param  {Boolean} silent Indicates to not show the global notice after the site is removed from the state.
+ * @return {Object}         Action object
  */
-export function receiveDeletedSite( siteId ) {
+export function receiveDeletedSite( siteId, silent = false ) {
```

Comment:
Would you like to add a quick test for is action creator? It'll be pretty straightforward.